lib/ansible/plugins/inventory/ovirt.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Red Hat, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: ovirt
+    plugin_type: inventory
+    short_description: oVirt inventory source
+    requirements:
+      - ovirt-engine-sdk-python >= 4.2.4
+    extends_documentation_fragment:
+        - inventory_cache
+        - constructed
+    description:
+      - Get inventory hosts from the ovirt service.
+      - Requires a YAML file ending in 'ovirt.yml', 'ovirt4.yml', 'ovirt.yaml', 'ovirt4.yaml'.
+    options:
+      plugin:
+        description: the name of this plugin, it should always be set to 'ovirt' for this plugin to recognise it as it's own.
+        required: True
+        choices: ['ovirt']
+      ovirt_url:
```

Comment:
Can this also support using the `OVIRT_URL` environment variable used by ovirt modules?
```
env:
  - name: OVIRT_URL
```